src/jit/lsra.h

```diff
@@ -647,6 +647,9 @@ class LinearScan : public LinearScanInterface
     // Used by Lowering when considering whether to split Longs, as well as by identifyCandidates().
     bool isRegCandidate(LclVarDsc* varDsc);
 
+    // Return the registers killed by the given tree node.
+    regMaskTP getKillSetForNode(Compiler* compiler, GenTree* tree);
```

Comment:
It was used in some debug assertions in an earlier version of the change. I've recombined the two overloads.